packages/indexer-client/src/types/serverTypes.ts

```diff
@@ -193,6 +194,13 @@ export interface IndexerServerLeaderboardContestsParams {
   contest_ids: number[];
 }
 
+/**
+ * @property {number | string} idx - The submission index of the WithdrawCollateral transaction to be used for fast withdraw.
```

Comment:
Btw - JS docs can be at the field level (i.e. a comment  right above L201), which I think works better with doc gen